tests/test_install_mender_sh.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python3
+# Copyright 2020 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import http.server
+import socketserver
+import threading
+import os
+import pytest
+
+SSERVER_PORT = 8000
```

Comment:
Is this a typo?
```suggestion
SERVER_PORT = 8000
```

Reply:
it was supposed to mean 'script server', but I guess not too intuitive.
I'll name it SCRIPT_SERVER, so that people won't be confused that we're running the mender server.